client/extensions/woocommerce/app/settings/shipping/shipping-classes-list.js

```diff
@@ -0,0 +1,176 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+
+import Button from 'components/button';
+import Card from 'components/card';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import ShippingClassEntry from './shipping-class-list-entry';
+import ShippingClassDialog from './shipping-class-dialog';
+
+import QueryShippingClasses from 'woocommerce/components/query-shipping-classes';
+import {
+	areShippingClassesLoaded,
+	areShippingClassesLoading,
+} from 'woocommerce/state/sites/shipping-classes/selectors';
+import { getUiShippingClasses } from 'woocommerce/state/ui/shipping/classes/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { addShippingClass } from 'woocommerce/state/ui/shipping/classes/actions';
+
+/**
+ * Renders the box for shipping class management.
+ */
+class ShippingClassesList extends Component {
+	static propTypes = {
+		onChange: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const { siteId, loaded, loading, translate } = this.props;
+
+		return (
+			<Fragment>
+				<QueryShippingClasses siteId={ siteId } />
+
+				<ExtendedHeader
+					label={ translate( 'Shipping Classes' ) }
+					description={ translate(
+						'Group products of similar type to provide different shipping ' +
+							'rates to each shipping class.'
```

Comment:
I'd suggest a tiny tweak to the copy: "[...] provide different shipping rates _for_ each shipping class."

Reply:
Tweaked.